packages/forms/resources/views/components/checkbox-list.blade.php

```diff
@@ -35,10 +39,28 @@
                 checkbox.dispatchEvent(new Event('change'))
             })
         },
+
+        search: '',
+
+        checkboxListItems: Array.from($root.querySelectorAll('label')),
+
+        visibleCheckboxListItems: [],
+
+        updateVisibleCheckboxListItems: function () {
+            this.visibleCheckboxListItems = this.checkboxListItems.filter((checkboxListItem) => {
+                return checkboxListItem.querySelector('span').innerText.toLowerCase().includes(this.search.toLowerCase())
+            })
+        }
+
     }">
         <div wire:key="{{ $this->id }}.{{ $getStatePath() }}.{{ $field::class }}.wrapper">
             @if ($isBulkToggleable() && count($getOptions()))
-                <div x-cloak class="mb-2" wire:key="{{ $this->id }}.{{ $getStatePath() }}.{{ $field::class }}.buttons">
+                <div
+                    x-cloak
```

Comment:
```suggestion
                    x-cloak=""
```

Reply:
This is only necessary when using x-cloak Blade components